mne_bids/dig.py

```diff
@@ -451,86 +419,103 @@ def _read_dig_bids(electrodes_fpath, coordsystem_fpath,
         Filepath of the electrodes.tsv to read.
     coordsystem_fpath : str
         Filepath of the coordsystem.json to read.
-    raw : mne.io.Raw
-        The data as MNE-Python Raw object.
     datatype : str
         Type of the data recording. Can be ``meg``, ``eeg``,
         or ``ieeg``.
+    raw : mne.io.Raw
+        The raw data as MNE-Python ``Raw`` object. Will set montage
+        read in via ``raw.set_montage(montage)``.
     verbose : bool
         Set verbose output to true or false.
 
     Returns
     -------
-    raw : mne.io.Raw
-        The data as MNE-Python Raw object.
+    montage : mne.channels.DigMontage
+        The coordinate data as MNE-Python DigMontage object.
     """
-    # get the space entity
-    params = get_entities_from_fname(electrodes_fpath)
-    space = params['space']
-    if space is None:
-        space = ''
-    space = space.lower()
-
     # read in coordinate information
-    coord_frame, coord_unit = _handle_coordsystem_reading(coordsystem_fpath,
-                                                          datatype, verbose)
+    bids_coord_frame, bids_coord_unit = _handle_coordsystem_reading(
+        coordsystem_fpath, datatype, verbose)
 
     if datatype == 'meg':
-        if coord_frame not in BIDS_MEG_COORDINATE_FRAMES:
+        if bids_coord_frame not in BIDS_MEG_COORDINATE_FRAMES:
             warn("MEG Coordinate frame is not accepted "
                  "BIDS keyword. The allowed keywords are: "
                  "{}".format(BIDS_MEG_COORDINATE_FRAMES))
             coord_frame = None
-        elif coord_frame == 'other':
+        elif bids_coord_frame == 'Other':
             warn("Coordinate frame of MEG data can't be determined "
                  "when 'other'. The currently accepted keywords are: "
                  "{}".format(BIDS_MEG_COORDINATE_FRAMES))
             coord_frame = None
         else:
-            coord_frame = BIDS_TO_MNE_FRAMES.get(coord_frame, None)
+            coord_frame = BIDS_TO_MNE_FRAMES.get(bids_coord_frame, None)
     elif datatype == 'ieeg':
-        if coord_frame not in BIDS_IEEG_COORDINATE_FRAMES:
-            warn("iEEG Coordinate frame is not accepted "
-                 "BIDS keyword. The allowed keywords are: "
-                 "{}".format(BIDS_IEEG_COORDINATE_FRAMES))
-            coord_frame = None
-        elif coord_frame == 'pixels':
+        # ieeg datatype for BIDS only supports
+        # acpc, pixels and then standard templates
+        # iEEG datatype for mne-python only supports
+        # mni_tal == fsaverage / MNI305
```

Comment:
```suggestion
        # mni_tal == fsaverage == MNI305
```
reduce potential for confusion ;)